db/migrate/20250325171006_add_sha256_to_claims_api_power_of_attorneys.rb

```diff
@@ -0,0 +1,5 @@
+class AddSha256ToClaimsApiPowerOfAttorneys < ActiveRecord::Migration[7.2]
+  def change
+    add_column :claims_api_power_of_attorneys, :sha256, :string
```

Comment:
We don't care what kind of hash algo was used. The whole reason we're required to make a new column right now is because the original column name indicated the algo used, and that name would be misleading if it started storing SHAs. Let's save ourselves from needing to do the same thing again should the algo change a second time.

Let's also have the field name indicate what it is a hash of. It's not well telegraphed right now - is it header data (`header_hash`)? Body data used for retrieval (`body_hash_id`)? A the lyrics to John Denver's "Country Road" (`take_me_home_hash`)?

Reply:
@cilestin I respectfully disagree on the reason for an additional column. We need the second column so we don't inadvertently disable our find function on older records. In addition, best DB practices dictate that instead of mixing data in one column, one should create a second. I have re-named the columns as you requested.